app/create/state.js

```diff
@@ -1,10 +1,12 @@
 'use strict';
 
+var HTML = require('../views/create.html');
```

Comment:
nit: if you're creating this file from scratch i'd use ES6 syntax (`const`, `() => `) etc

Reply:
Spoke IRL.